mlflow/tracking/client.py

```diff
@@ -396,6 +396,39 @@ def print_experiment_info(experiments):
             view_type=view_type, max_results=max_results, page_token=page_token
         )
 
+    def search_experiments(
+        self,
+        view_type: int = ViewType.ACTIVE_ONLY,
+        max_results: Optional[int] = None,
+        filter_string: Optional[str] = None,
+        order_by: Optional[List[str]] = None,
+        page_token=None,
+    ) -> PagedList[Experiment]:
+        """
+        Search for experiments that match the specified search criteria.
+
+        :param view_type: Qualify requested type of experiments.
```

Comment:
Can we be a bit more descriptive / specific about what this means? For example, here's the `view_type` docstring for the `search_runs` fluent API:

```
one of enum values ACTIVE_ONLY, DELETED_ONLY, or ALL runs defined in [mlflow.entities.ViewType](https://mlflow.org/docs/latest/python_api/mlflow.entities.html#mlflow.entities.ViewType).
```

Can we make this docstring change throughout the PR?

Reply:
Updated!